scripts/fix_homebrew_paths_in_standalone_zip.py

```diff
@@ -55,7 +54,7 @@ def readDependencies(fname):
                         command = "install_name_tool -change " + \
                             library + " @executable_path/./" + \
                             os.path.basename(library) + " " + fname
-                        print command
+                        print(command)
```

Comment:
When is this script run?

Reply:
It is called from `release.sh`, if run on OSX.